app/templates/components/settings/danger-zone.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+  {{#if doesUserHaveEventsOrOrders}}
+    <p>
+      {{t 'Your account currently cannot be deleted as events are associated with it. 
+      Please transfer the ownership of your events to another organizer.'}}
+    </p>
+    {{#ui-popup class='ui red button disabled'}}
+      Delete Your Account <i class="trash outline icon"></i>
+    {{/ui-popup}}
+  {{else}}
+    <p>
+      {{t 'All user data will be deleted. Your user data will be entirely erased and any data that will stay 
+      in the system for accounting purposes will be anonymised and there will be no link to any of your personal information. 
+      Ones you delete this account you will have no longer access to the system.'}}
+    </p>
+    {{#ui-popup click=(action 'openDeleteUserModal' data.id data.email) class='ui red button'}}
```

Comment:
Make this an icon button or a basic button maybe.

Reply:
@ritikamotwani I've made the required changes. Please review